documentation/writing_yaml_plans.md

```diff
@@ -124,6 +124,34 @@ prints the object as a string.
 For information on printing a step result with `message`, see [Debugging
 plans](#debugging-plans).
 
+
+### Verbose step
+
+Use a `verbose` step to print a message in verbose mode. The step prints a message to standard
+out (stdout) when using the `human` output format, and prints to standard error
+(stderr) when using the `json` output format. 
+
+Verbose steps support the following keys:
+
+| Key | Type | Description | Required |
+| --- | --- | --- | --- |
+| `verbose` | `String` | The message to print. | ✓ |
+
+For example:
+
+```yaml
+steps:
+  - verbose: hello world
+```
+
+You can pass variables to the verbose step to print them to stdout. If the
+variable contains a valid plan result, Bolt formats the plan result using a JSON
+representation of the result object. If the object is not a plan result, Bolt
+prints the object as a string.
+
+For information on printing a step result with `message`, see [Debugging
+plans](#debugging-plans).
```

Comment:
Also planned on deleting this, unless there is a useful debugging verbose link to point to?

Reply:
We can probably add using the `verbose` step [here](https://puppet.com/docs/bolt/latest/writing_yaml_plans.html#using-the---verbose-option-for-debugging).

Reply:
Hm, I think we could but I don't think we have to. The purpose of the function for debugging is mostly to be able to print variables, results, etc. I made a separate issue for pulling the debugging and logging in plans docs into their own page https://github.com/puppetlabs/bolt/issues/2933 and I think we can add `verbose` to the debugging page then, but for now I think we should just remove this line from the verbose step section.